.github/workflows/rav1e.yml

```diff
@@ -233,13 +238,28 @@ jobs:
 
     env:
       RUST_BACKTRACE: 1
+      RUSTC_WRAPPER: sccache
+      SCCACHE_CACHE_SIZE: 400M
```

Comment:
I propose to change the cache size here too in order to maintain the coherence with the `unix` version